lib/content-services/src/lib/tag/tag.module.ts

```diff
@@ -34,19 +36,12 @@ import { ContentDirectiveModule } from '../directives/content-directive.module';
         MaterialModule,
         FormsModule,
         ReactiveFormsModule,
-        CoreModule
+        CoreModule,
```

Comment:
the should not be core module here, can we avoid this somehow and import just what is needed? I suspect it's the TranslateModule

Reply:
Replaced dependency of core.module with translate.module and DynamicChipsListModule